README.md

```diff
@@ -310,13 +310,11 @@ $ composer require meilisearch/meilisearch-php php-http/curl-client nyholm/psr7:
 ```
 
 - with `kriswallsmith/buzz`, run:
--
+
 ```bash
 $ composer require meilisearch/meilisearch-php kriswallsmith/buzz nyholm/psr7:^1.0
 ```
 
```

Comment:
Maybe add the guzzle7 installation instruction too?

Reply:
That's a great idea, gone ahead and added!